src/sentry/static/sentry/app/utils/replaceRouterParams.tsx

```diff
@@ -1,4 +1,12 @@
-export default function replaceRouterParams(route, params) {
+/**
+ * Given a route string, replace path parameters (e.g. `:id`) with value from `params`
+ *
+ * e.g. {id: 'test'}
+ */
+export default function replaceRouterParams(
+  route: string,
+  params: {[key: string]: string}
```

Comment:
Does react-router have a params type we could use?

Reply:
No because this is not a `react-router` construct